pages/k8s/install-manual.md

```diff
@@ -3,112 +3,209 @@ wrapper_template: "base_docs.html"
+ </div>
+ <div class="col-4">
+   <span class="u-vertically-center"><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/gcp-overlay.yaml" class="p-button--positive">Download gcp-overlay.yaml</a></span>
+ </div>
+</div>
+<br>
+ <div class="row">
+  <div class="col-2">
+   <span>OpenStack integrator</span>
+ </div>
+  <div class="col-6">
+   <span>Provides support for OpenStack native features such as Cinder volumes and LBaaS. <a href="#"> Read more...</a></span>
+  </div>
+  <div class="col-4">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/openstack-overlay.yaml" class="p-button--positive u-no-margin--right">Download openstack-overlay.yaml</a></span>
+  </div>
+</div>
+<br>
+ <div class="row">
+  <div class="col-2">
+   <span>vSphere integrator</span>
+ </div>
+  <div class="col-6">
+   <span>Provides support native storage in vSphere. <a href="#"> Read more...</a></span>
+  </div>
+  <div class="col-4">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/vsphere-overlay.yaml" class="p-button--positive u-no-margin--right u-no-margin--left">Download vsphere-overlay.yaml</a></span>
+  </div>
+</div>
+</div>
 
-### Create a model
 
-The controller automatically creates a model named 'default'. It is useful to
-have a different model for each deployment of **CDK**, and for the models to have
-useful names. You can create a new model with the `add-model` command like
-this:
 
-```bash
-juju add-model cdk-test
-```
 
-...susbstituting the desired name.
 
-### Deploy the CDK bundle
 
-The **Juju Charm Store** hosts the **CDK** bundles as well as individual charms.  To
-deploy the latest, stable bundle, run the command:
+You can use multiple overlays (of different types) if required.  Note that all
+the 'integrator' charms require the use of the `--trust` option. For example,
+to deploy with Calico networking and AWS integration:
 
 ```bash
-juju deploy charmed-kubernetes
+juju deploy charmed-kubernetes --overlay aws-overlay.yaml --trust --overlay calico-overlay.yaml
 ```
 
-To get the status of the deployment, run `juju status`. For constant updates,
-combine it with the `watch` command:
+For more detail on overlays and how they work, see the section [below](#overlay).
 
-```
-watch -c juju status --color
+<a href="/kubernetes/docs/operations">Get started with your new cluster&nbsp;›</a>
+
+## Deploying a specific Charmed Kubernetes bundle
+
+The **Juju Charm Store** hosts the **Charmed Kubernetes** bundles as well as
+individual charms. To deploy the latest, stable bundle, run the command:
+
+```bash
+juju deploy charmed-kubernetes
 ```
 
 It is also possible to deploy a specific version of the bundle by including the
-revision number. For example, to deploy the **CDK** bundle for the Kubernetes 1.13
+revision number. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.13
```

Comment:
Do we want to update this since 1.13 will fall off support soon?

Reply:
Agree, let's make this 1.14.